src/_pytest/fixtures.py

```diff
@@ -1246,12 +1262,14 @@ def __init__(
     def __repr__(self) -> str:
         return f"<pytest_fixture({self._fixture_function})>"
 
-    def __get__(self, instance, owner=None):
+    def __get__(
+        self, obj: object, objtype: type | None = None
```

Comment:
Why deviate from The instance/owner pattern which is part of the stdlib docs

Reply:
Indeed, I must have looked at the wrong docs, will revert it.